src/features/commands.ts

```diff
@@ -638,47 +638,119 @@ Instead:
-            CREATE_PUBLIC_THREADS: false,
-            CREATE_PRIVATE_THREADS: false,
-            SEND_MESSAGES_IN_THREADS: false,
-          },
-        );
-        guildTextChannel.send("This channel has been locked by a moderator");
+        await msg.react("💩")
+        await msg.channel.send({
+          embeds: [
+            {
+              title: "tsk tsk",
+              type: "rich",
+              description: `${msg.author.toString()} This command is reserved for staff, Dont try to do this kind of shit`,
+              color: EMBED_COLOR,
+            },
+          ],
+        })
+
+      } else {
+        // permission overwrites can only be applied on Guild Text Channels
+        if (msg.channel.type === "GUILD_TEXT") {
+          const { channel: guildTextChannel } = msg;
+          await guildTextChannel.permissionOverwrites.create(
+            guildTextChannel.guild.roles.everyone,
+            {
+              SEND_MESSAGES: false,
+              CREATE_PUBLIC_THREADS: false,
+              CREATE_PRIVATE_THREADS: false,
+              SEND_MESSAGES_IN_THREADS: false,
+            },
+          );
+          guildTextChannel.send("This channel has been locked by a moderator");
+        }
       }
     },
   },
   {
     words: ["!unlock"],
-    help: "",
-    category: "Communication",
+    help: "used for unlocking channel",
+    category: "Moderation",
     handleMessage: async (msg) => {
       if (!msg.guild || !isStaff(msg.member)) {
-        return;
+        await msg.react("💩")
+        await msg.channel.send({
+          embeds: [
+            {
+              title: "tsk tsk",
+              type: "rich",
+              description: `${msg.author.toString()} This command is reserved for staff, Dont try to do this kind of shit`,
+              color: EMBED_COLOR,
+            },
+          ],
+        })
+      } else {
+        // permission overwrites can only be applied on Guild Text Channels
+        if (msg.channel.type === "GUILD_TEXT") {
+          const { channel: guildTextChannel } = msg;
+          await guildTextChannel.permissionOverwrites.create(
+            guildTextChannel.guild.roles.everyone,
+            {
+              SEND_MESSAGES: null, // null will inherit the permission from the parent channel category
+            },
+          );
+        }
       }
-
-      // permission overwrites can only be applied on Guild Text Channels
-      if (msg.channel.type === "GUILD_TEXT") {
-        const { channel: guildTextChannel } = msg;
-        guildTextChannel.permissionOverwrites.create(
-          guildTextChannel.guild.roles.everyone,
-          {
-            SEND_MESSAGES: null, // null will inherit the permission from the parent channel category
-          },
-        );
+    },
+  },
+  {
+    words: ["!ban"],
+    help: "used for banning users \n\t(usage: `!ban reason user`)",
+    category: "Moderation",
+    handleMessage: (msg) => {
+      if (!msg.guild || !isStaff(msg.member)) {
```

Comment:
**Note:** This would also allows MVPs to ban, which is different from our Discord permissions

Reply:
So, is it ok ??? Or should I change behaviour of it ??

Reply:
@vcarl and @nickmccurdy what should i do ?? or it is ok ??